WARNING  root:Logger.py:10 firefox
WARNING  root:Logger.py:10 =======================SEPARATOR=================================
WARNING  root:Logger.py:10 firefox
WARNING  root:Logger.py:10 2
WARNING  root:Logger.py:10 __Тест на ввод Логина__
WARNING  root:Logger.py:10 <selenium.webdriver.firefox.webdriver.WebDriver (session="b7de2cc6-bff3-4470-a539-294820054489")>
WARNING  root:Logger.py:10 Главная страница открыта
WARNING  root:Logger.py:10 __Тест на ввод неправильного логина и пароля__
WARNING  root:Logger.py:10 <selenium.webdriver.firefox.webdriver.WebDriver (session="b7de2cc6-bff3-4470-a539-294820054489")>
WARNING  urllib3.connectionpool:connectionpool.py:812 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B759DD2140>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение')': /session/b7de2cc6-bff3-4470-a539-294820054489/url
WARNING  urllib3.connectionpool:connectionpool.py:812 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B759DD1ED0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение')': /session/b7de2cc6-bff3-4470-a539-294820054489/url
WARNING  urllib3.connectionpool:connectionpool.py:812 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001B759DD1E40>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение')': /session/b7de2cc6-bff3-4470-a539-294820054489/url
WARNING  root:Logger.py:10 firefox
WARNING  root:Logger.py:10 =======================SEPARATOR=================================
WARNING  root:Logger.py:10 firefox
WARNING  root:Logger.py:10 2
WARNING  root:Logger.py:10 __Тест на ввод правильного логина и пароля__
WARNING  root:Logger.py:10 <selenium.webdriver.firefox.webdriver.WebDriver (session="453a24b2-c7f9-43f0-b69c-19536396d8d3")>
WARNING  root:Logger.py:10 Главная страница открыта
